resources/public/css/main.css

```diff
@@ -468,9 +468,23 @@ Ensure the default browser behavior of the `hidden` attribute.
 .relative {
   position: relative;
 }
+.mx-auto {
+  margin-left: auto;
+  margin-right: auto;
+}
+.my-4 {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
 .mr-2 {
   margin-right: 0.5rem;
 }
+.mb-2 {
+  margin-bottom: 0.5rem;
+}
+.mb-4 {
+  margin-bottom: 1rem;
+}
```

Comment:
It's cool that this file updates based on the classes that are used. Nice overview of styling.

Reply:
even though locally generated files probably shouldn't be committed huh?

Reply:
yeah i didnt add a gitignore for this.. probably want to remove it from github